package.json

```diff
@@ -50,7 +50,8 @@
   },
   "dependencies": {
     "@babel/preset-env": "^7.10.4",
-    "uuid": "^3.3.3"
+    "uuid": "^3.3.3",
+    "yeoman-generator": "^4.12.0"
```

Comment:
Shouldn't this be in devDependencies? Consumers of this package don't need this to ship to customers :)

Reply:
yes agreed! Misplaced it.